cmd/attack/process.go

```diff
@@ -62,6 +62,7 @@ func NewProcessKillCommand(dep fx.Option, options *core.ProcessCommand) *cobra.C
 
 	cmd.Flags().StringVarP(&options.Process, "process", "p", "", "The process name or the process ID")
 	cmd.Flags().IntVarP(&options.Signal, "signal", "s", 9, "The signal number to send")
+	cmd.Flags().StringVarP(&options.RecoverCmd, "recover-cmd", "r", "", "The command to be executed at the end of experiment")
```

Comment:
how about changing the description to `The command to be executed when recovering experiment`?

Reply:
I agree with you, your description is more appropriate.